nasl/nasl_global_ctxt.h

```diff
@@ -36,6 +37,9 @@ typedef struct
 int
 init_nasl_ctx (naslctxt *, const char *);
 
+int
+init_notus_vt_list (const kb_t, const char *, int);
```

Comment:
I would not use the name notus. Why not something like init_csv_vt_list ?
Or even just init_csv_file or init_csv_file_check.